config.m4

```diff
@@ -3,8 +3,8 @@ PHP_ARG_ENABLE(mongodb, whether to enable mongodb support,
 [  --enable-mongodb           Enable mongodb support])
 PHP_ARG_WITH(openssl-dir, OpenSSL dir for mongodb,
 [  --with-openssl-dir[=DIR]  openssl install prefix], yes, no)
-
-
+PHP_ARG_WITH(system-ciphers, whether to use system default cipher list instead of hardcoded value,
+[  --with-system-ciphers   OPENSSL: Use system default cipher list instead of hardcoded value], no, no)
```

Comment:
This is the same argument name used in the OpenSSL extension's `config0.m4`. I assume these should be kept in sync for cases where the extension is compiled into PHP directly (just as our `openssl-dir` arg syncs up with the one used for the FTP extension).